### DISCUSSION TITLE
Can the library do a transaction across tables?

### DISCUSSION THREAD

**Discussion by sweepy84** [sensedeep/dynamodb-onetable#493] [Q&A] [answered]
DynamoDB provides for transactions across tables, but I can't seem to find in the one-table documentation an example of this. Is this possible?

CC @axsj

**Comment by mobsense** [comment, marked as answer]
OneTable does not support cross-table transactions (yet).

Sorry.

**Reply by sweepy84** [reply]
Ok thanks. Would be good to add to the documentation to avoid confusion.

Any plans on when this feature will be brought in. It's one of those features you don't need until you do.

**Reply by mobsense** [reply]
No definite plans. I'd recommend you open a bug/feature. Then it will be prominent and "haunt" us till it is implemented.
